cobra/flux_analysis/loopless.py

```diff
@@ -221,11 +221,11 @@ def loopless_fva_iter(model, reaction, solution=False, zero_cutoff=1e-6):
         # almost loopless solution containing only loops including the current
         # reaction. Than remove all of those loops.
         ll_sol = get_solution(model).fluxes
-        bounds = reaction.bounds
         reaction.bounds = (current, current)
         model.slim_optimize()
         almost_ll_sol = get_solution(model).fluxes
-        reaction.bounds = bounds
+        print(ll_sol, almost_ll_sol)
```

Comment:
Could you remove this again, please? I suppose it was for debugging. Otherwise maybe introduce a logger here.

Reply:
Ouch, sorry. fixed now.